view/ce_dashboard/flask/requirements.txt

```diff
@@ -1,6 +1,6 @@
 htcondor
 numpy<2
-filelock==3.18.0
+filelock==3.20.1
```

Comment:
Note sure what best practice is here, but do we want to force `filelock` to exactly 3.20.1, or would we be willing to accept any 3.20.x version?